src/envo/envo-edit.owl

```diff
@@ -17007,6 +17008,17 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_00009726> "strand crack"^^xsd:string)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00009726> <http://purl.obolibrary.org/obo/ENVO_01000667>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_00010005> (soil bulk density)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00010005> "A volumetric density measurement which inheres in soil by virtue of some influence exerted by the soil's mass on a given volume."@en)
```

Comment:
A measurement can't inhere in something, but a quality / characteristic can. This needs some more semantic unpacking, I think.